### ISSUE TITLE
About custom image inference code

### ISSUE THREAD

**Issue by park-youngjae** [waxnkw/IETrans-SGG.pytorch#5]
Thank you for your wonderful work!

Is there any plan to upload code about custom image inference code(Predcls, SGCls, SGDet)?

Thanks.

**Comment by waxnkw** [comment]
Thanks for your interest! I am working on using SGG model for other dataset's relation detection. I think I will write this code before September. If the code is updated, I will inform you through this issue.
Meanwhile, I think you can try to do it with the reference to https://github.com/KaihuaTang/Scene-Graph-Benchmark.pytorch/issues?q=custom. The basic way of using our code is similar to theirs.

**Comment by park-youngjae** [comment]
Thank you for your quick response. 

I'll be looking forward to your code update.

**Comment by park-youngjae** [comment]
Thank you for your quick response. 

I'll be looking forward to your code update.

**Comment by waxnkw** [comment]
Updated. Please refer to [DEMO.md](https://github.com/waxnkw/IETrans-SGG.pytorch/blob/master/DEMO.md).

**Comment by park-youngjae** [comment]
Thank you for responding to my feedback!
Is there a demo for precls task too?

**Comment by waxnkw** [comment]
Sorry, I do not add the predcls demo because normally it is hard to get object labels exactly matched with VG's classes from other detectors (like COCO pre-trained).

If you have a special requirement and do not want to modify code, you can directly use SGCLS mode and modify the predicted labels to your own after getting the scene graph.

If you want to modify code, you can simply modify one line at [here](https://github.com/waxnkw/IETrans-SGG.pytorch/blob/ddf13cbfeae4e17ee670acf9e210d8c92f93e7d9/maskrcnn_benchmark/data/datasets/visual_genome.py#L156) to add your own labels.
